model/database/cosmosdb-metrics.yaml

```diff
@@ -0,0 +1,48 @@
+groups:
+  - id: metric.db.cosmosdb.operation.request_charge
+    type: metric
+    metric_name: db.cosmosdb.operation.request_charge
+    brief: "[Request charge](https://learn.microsoft.com/azure/cosmos-db/request-units) consumed by the operation"
+    instrument: histogram
+    unit: "{request_unit}"
+    stability: experimental
+    extends: attributes.db.cosmosdb.minimal
+
+  - id: metric.db.cosmosdb.client.active_instances
+    type: metric
+    metric_name: db.cosmosdb.client.active_instances
+    brief: "Number of active client instances"
+    instrument: histogram
+    unit: "{instance}"
+    stability: experimental
+    extends: attributes.db.cosmosdb.minimal
+
+  - id: attributes.db.cosmosdb.minimal
```

Comment:
you're loosing the clarifications made for cosmos DB span with this.

the suggestion:

1. take the clarifications (levels, briefs, notes) of applicable attributes from  `db.cosmosdb` group defined here https://github.com/open-telemetry/semantic-conventions/blob/76112ddf3a86d6e51e2366a6a2039db2db567466/model/database/spans.yaml#L516
2. put the resulting `attributes.db.cosmosdb.minimal` group to `database/common.yaml`
3. extend `attributes.db.cosmosdb.minimal` group in cosmos db spans. You might need to play a bit with extension to make spans look exactly the same, but it should be doable.